frontend/src/components/AdminControls/Users.js

```diff
@@ -1,133 +1,216 @@
-import React, { Fragment } from 'react';
+import React, { Component, Fragment } from 'react';
 import BootstrapTable from 'react-bootstrap-table-next';
 import paginationFactory from 'react-bootstrap-table2-paginator';
 import { Type } from 'react-bootstrap-table2-editor';
 import './AdminTables.css';
+import AdminEditModal from '../AdminEditModal/AdminEditModal';
+import AdminSuspendModal from '../AdminModal/AdminSuspendModal';
 
-const Users = ({ users }) => {
-  const columns = [
-    {
-      dataField: 'inStock', // name?
-      text: '',
-      headerStyle: {
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-      style: {
-        paddingLeft: '10px',
-      },
-      formatter: (_cellContent, row) => (
-        <div className="checkbox">
-          <label htmlFor="adminCheckbox" className="admin-checkbox-container user-checkbox-spacing">
-            <input type="checkbox" id="adminCheckbox" checked={row.inStock} />
-            <span className="admin-table-checkmark" />
-          </label>
-        </div>
-      ),
-    },
-    {
-      dataField: 'full_name',
-      text: 'User Name',
-      style: { textAlign: 'center' },
-      headerStyle: {
-        textAlign: 'center',
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-    },
-    {
-      dataField: 'created_at',
-      text: 'Member Since',
-      style: { textAlign: 'center' },
-      headerStyle: {
-        textAlign: 'center',
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-      formatter: (cell) => {
-        let dateObj = cell;
-        if (typeof cell !== 'object') {
-          dateObj = new Date(cell);
-        }
+// add selectedUser to state which will update based on rowEvents click
+// then the modal will open on button click, and can access selected user in state
 
-        const day = `0${dateObj.getUTCDate()}`.slice(-2);
-        const month = `0${dateObj.getUTCMonth() + 1}`;
-        const year = dateObj.getUTCFullYear();
-        return `${month}/${day}/${year}`;
-      },
-      editor: {
-        type: Type.DATE,
+class Users extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      editModal: false,
+      suspendModal: false,
+      selectedUser: {},
+    };
+    this.toggleEditModal = this.toggleEditModal.bind(this);
+    this.toggleSuspendModal = this.toggleSuspendModal.bind(this);
+  }
+
+  toggleEditModal() {
+    this.setState(prevState => ({
+      editModal: !prevState.editModal,
+    }));
+  }
+
+  toggleSuspendModal() {
+    this.setState(prevState => ({
+      suspendModal: !prevState.suspendModal,
+    }));
+  }
+
+  render() {
+    const { users } = this.props;
+    const columns = [
+      {
+        dataField: 'inStock', // name?
```

Comment:
Everything here seems fine. Maybe take out the comment?

Reply:
Does the comment count as "documentation"?